resources/sass/coba-ci.scss

```diff
@@ -182,6 +182,9 @@
     &.coba-button-accent{
         background-color: $coba-yellow;
     }
+    &.coba-button.accent{
```

Comment:
diese Zeile brauchst du nicht. 
Erstens ist coba-button sowieso schon ausgwählt durch das "&". Deswegen würdest du dann schreiben "&.accent"
Allerdings haben wir direkt darüber in Zeile 183 doch schon einen coba-button-accent definiert, den du verwenden kannst ;)

Reply:
keine ahnung was ich da gemacht.....😂